tables/tr-g2.ctb

```diff
@@ -0,0 +1,599 @@
+﻿# liblouis: Türkçe Tam Kısaltmalı Braille
+#
+#  Copyright (C) 2009 SBS Schweizerische Bibliothek für Blinde und Sehbehinderte

```

Comment:
Why is the copyright with SBS? Presumably it lies with you @simait or your employer.